src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -16,4 +16,5 @@
     public static final Prefix PREFIX_TASKNAME = new Prefix("tn/");
     public static final Prefix PREFIX_DATETIME = new Prefix("dt/");
     public static final Prefix PREFIX_PERSON = new Prefix("p/");
+    public static final Prefix PREFIX_LINK = new Prefix("z/");
```

Comment:
Maybe there can be a better prefix then z/ to better reflect the purpose of this field